test/hotspot/jtreg/containers/docker/TestMemoryAwareness.java

```diff
@@ -153,18 +153,24 @@ private static void testOperatingSystemMXBeanAwareness(String memoryAllocation,
         out.shouldHaveExitValue(0)
            .shouldContain("Checking OperatingSystemMXBean")
            .shouldContain("OperatingSystemMXBean.getTotalPhysicalMemorySize: " + expectedMemory)
-           .shouldMatch("OperatingSystemMXBean\\.getFreePhysicalMemorySize: [1-9][0-9]+")
            .shouldContain("OperatingSystemMXBean.getTotalMemorySize: " + expectedMemory)
            .shouldMatch("OperatingSystemMXBean\\.getFreeMemorySize: [1-9][0-9]+")
-           .shouldMatch("OperatingSystemMXBean\\.getFreeSwapSpaceSize: [1-9][0-9]+");
-        // in case of warnings like : "Your kernel does not support swap limit capabilities or the cgroup is not mounted. Memory limited without swap."
-        // the getTotalSwapSpaceSize returns the system values as the container setup isn't supported in that case.
+           .shouldMatch("OperatingSystemMXBean\\.getFreePhysicalMemorySize: [1-9][0-9]+");
+
+        // in case of warnings like : "Your kernel does not support swap limit capabilities
+        // or the cgroup is not mounted. Memory limited without swap."
+        // the getTotalSwapSpaceSize and getFreeSwapSpaceSize return the system
+        // values as the container setup isn't supported in that case.
         try {
             out.shouldContain("OperatingSystemMXBean.getTotalSwapSpaceSize: " + expectedSwap);
         } catch(RuntimeException ex) {
-            out.shouldMatch("OperatingSystemMXBean.getTotalSwapSpaceSize: [1-9][0-9]+");
-            out.shouldContain("Metrics.getMemoryLimit() == " + expectedMemory);
-            out.shouldContain("Metrics.getMemoryAndSwapLimit() == -1");
+            out.shouldMatch("OperatingSystemMXBean.getTotalSwapSpaceSize: [0-9]+");
```

Comment:
This should probably be:
```
out.shouldMatch("OperatingSystemMXBean\\.getTotalSwapSpaceSize: [0-9]+");
```

Reply:
Heh, it was pre-existing :)

Reply:
This isn't critical for this patch. As an pre-existing issue I won't insist on this. Either way is fine for me.